libs/core/langchain_core/messages/utils.py

```diff
@@ -1424,3 +1425,72 @@ def _convert_to_openai_tool_calls(tool_calls: list[ToolCall]) -> list[dict]:
         }
         for tool_call in tool_calls
     ]
+
+
+DEFAULT_APPROXIMATE_TOKEN_LENGTH = 4
+DEFAULT_EXTRA_TOKENS_PER_MESSAGE = 3
+
+
+def count_tokens_approximately(
+    messages: list[BaseMessage],
+    token_length: int = DEFAULT_APPROXIMATE_TOKEN_LENGTH,
+    extra_tokens_per_message: int = DEFAULT_EXTRA_TOKENS_PER_MESSAGE,
```

Comment:
Probably float this one as well

```suggestion
    extra_tokens_per_message: float = DEFAULT_EXTRA_TOKENS_PER_MESSAGE,
```